tools/cockpit.spec

```diff
@@ -272,7 +272,6 @@ cat kdump.list subscriptions.list sosreport.list networkmanager.list selinux.lis
 %package bridge
 Summary: Cockpit bridge server-side component
 Requires: glib-networking
-Requires: polkit
```

Comment:
tools/debian/control should then get the same, lowering `Depends: policykit-1` to `Suggests:`.